resources/brick/brick_instance.ttl

```diff
@@ -131,12 +130,12 @@ bldg:chiller a brick:Chiller ;
 
 bldg:chiller_chilled_water_flow_sensor a brick:Chilled_Water_Flow_Sensor .
 
-bldg:chiller_chilled_water_return_temperature_sensor a brick:Chilled_Water_Return_Temperature_Sensor .
```

Comment:
This class was deprecated in brick v1.3 and thus replaced with the new class ```Leaving_Water_Temperature_Sensor```. The below changes are the same.

Reply:
Thanks Yun Jong. Is this the reason for validation failure we saw earlier?